package.json

```diff
@@ -12,6 +12,8 @@
     "start:eleventy": "eleventy --serve",
     "start:postcss": "postcss src/styles/*.css --dir _site --watch",
     "format": "npx eslint --fix _site/audio-worklet/**/*.js && npx prettier --write --loglevel silent _site/audio-worklet/**/*.html",
+    "lint:prettier": "prettier --check src/**/*.js src/**/*.html",
```

Comment:
If we remove the "Lint changed files" step we may want to add the other relevant filetypes here.

Reply:
I decided to keep the "Lint changed files" step, which runs linters only on the changed files. As a result, there's no need to modify the package.json file to include additional file types for linting, since the dynamic filtering for relevant files is handled in the "Lint changed files" step itself.

Reply:
Can we remove the "lint:prettier" and "lint:eslint" lines now too?

Reply:
Yes, removed both "lint:prettier" and "lint:eslint" from `packagejson`